pkg/version/version.go

```diff
@@ -99,6 +99,13 @@ func GetVersionBundle(version string) (VersionBundle, error) {
 		MulticlusterControlplane: "v0.5.0",
 	}
 
+	versionBundleList["0.15.0"] = VersionBundle{
+		OCM:                      "v0.15.0",
```

Comment:
@dhaiducek @mikeshng @clyang82 wolud you confirm on the version update here?

Reply:
Yes, I just tagged GRC/Policy for v0.15.0 and submitted a manifest update PR here:
- https://github.com/open-cluster-management-io/clusteradm/pull/452